Buildings/Resources/Scripts/Dymola/ConvertBuildings_from_7_to_8.0.0.mos

```diff
@@ -74,3 +74,8 @@ convertClass(
 convertClass(
     "Buildings.Controls.OBC.CDL.Types.Reset",
     "Buildings.Obsolete.Controls.OBC.CDL.Types.Reset");
+
+// Conversion for https://github.com/lbl-srg/modelica-buildings/issues/2172
+// remove parameter V_flow_nominal
+convertModifiers("Buildings.Controls.OBC.ASHRAE.G36_PR1.TerminalUnits.Controller",
+                {"V_flow_nominal"}, fill("",0), true);
```

Comment:
This command will delete the parameter binding for `V_flow_nominal`. However, the converted model will still fail to simulate as the air flow set point parameters are bound by default to `V_flow_nominal` in the implementation on the master. Additionally, the user will lose the information about the original value that was assigned to `V_flow_nominal`, which is more problematic.
A call like the following would remove `V_flow_nominal` and assign the current default values to the other set points.
```
convertModifiers("Buildings.Controls.OBC.ASHRAE.G36_PR1.TerminalUnits.Controller",
    {"V_flow_nominal", "VDisConMin_flow"},
    {"VDisConMin_flow=0.1*%V_flow_nominal%"},
    true);
```
However, I cannot see a call to `convertModifiers` that would preserve a possible overriding from the user, for instance if he changed the default to `VDisConMin_flow = 0.2 * V_flow_nominal` or even a literal value.

Reply:
I agree to add the script as you suggested, to update the default values of the set points. So far, I also cannot see how to solve the issue when the user made the change to the default values.
I suggest that we just update the default values.